app/templates/common/Gruntfile.js

```diff
@@ -8,29 +8,16 @@ module.exports = function (grunt) {
         configDir: require('path').resolve('tasks')
     });
 
-    // Register group tasks
-    grunt.registerTask('build', [ 'jshint', <%
-        if (cssModule) {
-            %>'<%= cssModule %>', <%
-        }
-
-        %><%
-
-        if (jsModule) {
-            %>'<%= jsModule %>', <%
-        }
+    <% if (templateModule === 'makara' && jsModule == 'browserify') { -%>
+    grunt.loadNpmTasks('grunt-makara-browserify');
+    <% } -%>
 
-        %><%
+    <% if (templateModule === 'makara' && jsModule == 'requirejs') { -%>
+    grunt.loadNpmTasks('grunt-makara-amdify');
+    <% } -%>
 
-        if (i18n) {
-            %>'i18n', <%
-        } else {
-            if (templateModule) {
-                %>'<%= templateModule %>', <%
-            }
-        }
-
-        %>'copyto' ]);
+    // Register group tasks
+    grunt.registerTask('build', <%- JSON.stringify(tasks) %>);
```

Comment:
True that. I was wondering if that would bother anyone -- what was there before was kinda terrible.